src/main/java/ro/msg/learning/shop/domain/OrderDetail.java

```diff
@@ -0,0 +1,32 @@
+package ro.msg.learning.shop.domain;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+import ro.msg.learning.shop.domain.primarykeys.OrderDetailKey;
+
+@Entity
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "order_detail")
+@IdClass(OrderDetailKey.class)
+public class OrderDetail {
+    @Id
+    @JdbcTypeCode(SqlTypes.UUID)
+    @Column(name = "order_id")
+    private Order orderId;
+
+    @Id
+    @JdbcTypeCode(SqlTypes.UUID)
```

Comment:
Is the `@JdbcTypeCode(SqlTypes.UUID)` really necessary? If not, please remove it from all the attributes.

Reference: https://thorben-janssen.com/generate-uuids-primary-keys-hibernate/